packages/hash/frontend/src/components/hooks/use-get-workspace-by-shortname.ts

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from "react";
+import { MinimalOrg } from "../../lib/org";
+import { User } from "../../lib/user";
+import { useOrgs } from "./useOrgs";
+import { useUsers } from "./useUsers";
+
+export const useGetWorkspaceByShortname = (
```

Comment:
Nit:


```suggestion
export const useWorkspaceByShortname = (
```

We return data, so omitting `Get` from the hook name makes its purpose a bit more intuitive. Otherwise, one may expect something like this:

```js
const getFoo = useGetFoo();
getFoo(someParam);
```

We don't have to address this now, but maybe as part of https://app.asana.com/0/1201095311341924/1202863271046362/f.

We can also move this hook somewhere else. It's currently in `components/hooks`, which may become a dump for all hooks at one point. Instead, we can see which 'feature' this hook belongs to, put it to `../(../)../shared/some-feature/*` and re-export from `../(../)../shared/some-feature.js`.

Reply:
I've renamed the hook as part of https://github.com/hashintel/hash/pull/1650/commits/fa2ee8b97d56b079aca29075b602fd6243d6819b